apps/consoledot-rhosak/src/routes/data-plane/routes/TopicCreateRoute.tsx

```diff
@@ -117,6 +117,11 @@ export const TopicCreateRoute: VoidFunctionComponent<
         };
   }, [instance.plan]);
 
+  const isSaving = (() => {
+    if (createTopic.isLoading) return true;
+    else return false;
+  })();
```

Comment:
```suggestion
  const isSaving = createTopic.isLoading;
```

No need for an IIFE for this...